build/tasks/lib/verifyNodeVersion.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+const { version } = require( "process" );
+const nodeV16OrNewer = !/^v1[0-5]\./.test( version );
+
+module.exports = function verifyNodeVersion() {
```

Comment:
Too bad we still need to run on Node 10 or we could write ESM even for Node.js...